docs/development.md

```diff
@@ -417,15 +417,16 @@ To run a custom test suite on a CAPZ cluster locally, set `AZURE_CLIENT_ID`, `AZ
 
 You can optionally set the following variables:
 
-| Variable                       | Description                                                                                                   |
-|--------------------------------|---------------------------------------------------------------------------------------------------------------|
-| `AZURE_SSH_PUBLIC_KEY_FILE`    | Use your own SSH key.                                                                                         |
-| `SKIP_CREATE_WORKLOAD_CLUSTER` | Skip workload cluster creation.                                                                               |
-| `SKIP_CLEANUP`                 | Skip deleting the cluster after the tests finish running.                                                      |
-| `KUBECONFIG`                   | Provide your existing cluster kubeconfig filepath. If no kubeconfig is provided, `./kubeconfig` will be used.     |
-| `USE_CI_ARTIFACTS`             | Use a CI version of Kubernetes, ie. not a released version (eg. `v1.19.0-alpha.1.426+0926c9c47677e9`)         |
-| `CI_VERSION`                   | Provide a custom CI version of Kubernetes. By default, the latest master commit will be used.                 |
-| `EXP_MACHINE_POOL`             | Use [Machine Pool](topics/machinepools.md) for worker machines.                                               |
+| Variable                       | Description                                                                                                                                |
+|--------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| `AZURE_SSH_PUBLIC_KEY_FILE`    | Use your own SSH key.                                                                                                                      |
```

Comment:
can you move this to https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/book/src/developers/development.md ? I need to delete this file again, looks like it got re-added in a rebase

Reply:
Deleted in this PR